libcloud/common/openstack_identity.py

```diff
@@ -926,8 +929,7 @@ class OpenStackIdentity_3_0_Connection(OpenStackIdentityConnection):
     ]
 
     def __init__(self, auth_url, user_id, key, tenant_name=None,
-                 domain_name='Default',
-                 token_scope=OpenStackIdentityTokenScope.PROJECT,
+                 domain_name=None, token_scope=None,
```

Comment:
Before this PR the only [place where it is called](https://github.com/apache/libcloud/pull/744/files#diff-cb42abf0c253c7c80a89e7a9293d15a3R199) wasn't passing these two parameters so their default value was the only possible value.
With the way this parameter is passed from the driver's `__init__` to this method, it will be None if it is omitted. 
So I changed the logic from
> use the default value if it is not passed during the instanciation of OpenStackIdentity_3_0_Connection

(which is a case that cannot happens)  
to 
> use the default value if it is None

This is also why I removed the `if` below.